### PR TITLE
Geolocation web message

### PR DESCRIPTION
**Pull request by ceoaliongroo**
#82

### CODE REVIEW

File: app/scripts/services/phonegap.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 angular.module('publicEducationApp')
-  .service('Phonegap', function Phonegap($http) {
+  .service('Phonegap', function Phonegap($http, $window) {
```
Comment (amitaibu) [inline app/scripts/services/phonegap.js]: Is this related?

**Comment by ceoaliongroo** [comment]
Merged.
